### ISSUE TITLE
explosive arrow - totem duration bug

### ISSUE THREAD

**Issue by Yikss** [PathOfBuildingCommunity/PathOfBuilding#6453]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the calculation is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the value from the calculation in-game?

in ingame it seems totem duration nod don't effect explosive arrow duration

### What is the value from the calculation in Path of Building?

at previous version, my explosive arrow  duration was 1.5+a second, after update however it became 2.5second.

pob says Primal Manifestation, sleepless Sentries, Totem place ment speed and duration nodes provides 50%, 20%, 30% duration to explosive arrow.

### How to reproduce the issue

just give explosive arrow blaista support, and then procide totem duration

### Character build code

```shell
eNrtfWlz47ay6OfoV7B86756r8YLAAIgOW_m3JKXsZ3YM47tmSxfUiAJWsxQpENSXubU-e-3G6QkUha1WM52ziQVRxLRDXSjdyx88z8Pw8S603kRZ-nbLbpLtiydBlkYpzdvtz5ev9txt_7nH703F6ocfIj2R3GCT_7R--aN-Wwl-k4nb7c8vmWVKr_R5acxKvsXQHWr0nKgs_Rc_Zrlx1n4dut9luoty1dpGJfjb0GiiuK9Guq3Wz_EZTDYslQR6DQ8mP5-lOihTkuVxEUJAAOVq6DU-Rl23x-V2XkWQqtIJQXgG6o4vcqCz7o8zrPRLZAFQ7mL9X3V6vT84sPldWNwcdocHBD3zZuLRD3q_KpUpVXAn7dbfeCRutGHagh_AZtKRoCKuswhlO86zGZia28h6P4oL8rnwF_dah1OQNgu5dTuansSl-3mZFdyZntd7a-zUg_h18Cw9zoe6gYkdTzuOvVftwvFRa6PokgHZXynD_K4PBioNJiikbukk7JnND8fJWV8m8Q6nzJxATNmsFNCFnBCJYcXV9O21LGlELue6xLpysVwWTnlW1fL05s0LnWzC5e4jDtylzFOiLCXALZFh0nuelLyXUKpcDonGKQuBY1FRUjDZt-MMCKpB-yWhHQL1Ji8FmcIJTbxxC5zCSFO53SNkgS6bkJ28uZSFzq_UyWYjpXaH2RDP0512L-7mVLk2lRKx911bO6yTo70c60-RJXIXqowHhVTDJ20nKtUHWTFCrOMLS90DvarXAvgSgcZmLx1-1gT8iyOdKulwxY1XYuQGmDOcDihYLWYKxz8K1focU2yjq5Wbbc24ucN6BJUdbWWV9koWbFl2bB5Qnar0m-tljbvanmoH6bNOF2AsN2y28KlZaNfZwHCdssFI7zL0CYsZ46xU0cnF1OsHtiAXTSrnFG30xpcDB6LOFDJuXqIh6MhuIxr9VmnDVvH5AKJvxmUaF47oV3pdPL1XZzrZwEeZEn4PMCByoouSC7ZQs2cNLQp4QtileA1tj5Ng6mNWWgyPqa5Mf6NoGVRDwhyCZqIQZWf6JVhpt3UGr1KUFB1dqPTusfHKZAUuwv7O9M6GBxDJArdAa9XM-rTMMhlC1mMjZssXoh1DofpigBPeLXLpWSOkALUiy_CMZ9tzNl1F0GtzbWjVOc3j1eDWCdT6lyxSvPx4A7U7QqQhu1N8Cb7KVutw3ksWYm0tfnyg8rD1dzUuqO6U0XTItuQHSzmWQXQZJfXLX060RogQj0TuXfHcxd59itmH8macP18mI3ypqF3FhNSATTp6KZ87FaqoP1Sh6Og5cdkp-7sJ5C5zhBBFowrSdaC6JelCj4fZuGNXquT9SGuRre3YEZw7mdzPL7IJUIqEDcDVLG88QcQ3aYWd4ce6DpX7mDaeLYDsTwkWLmXGYgntJCF3nymmx26SvPZPrqVfZzYn4N1wAqBKS6cZw1Hstvtqd5BAtjOqJkjHUF3mXTJgnlFsFUyzhr_1ec4SawC_16oHBBsWQUoaoBlo4fbJCtg_FY_z7P7rbGO6aK0qqzaWp6Pb9ZNnYBbVQZuLU_Bu7qrO8lS6__-l5VFVjQqdPH_poMY9_pkMOvVMy6yexCOARbziuUC0mwNgeVUajsnN9fpl8dZ_Ks1X6mDozQc5WhuVqZhFmKlbvpGLUz57CweNjzymLFYS0vA_B2qUllhncd8Unms0pJh0coqtMqDwRno4zuVJD6YZTOJk18rCcMy5rs4KXV-CL_hMJGUWYx0XAZ7s2cKs_jpdHib5aWVqKLsBwFIX3miisHbLZ95trJ9RwhqR8RTgeTS9VyHq8gPpU996pHQ86ItA3qpVTKEiT7YsvQDIkRpfJzUWrHJwbgcW-Hn2lGOFnbgMMfxNI04t6kThZEQygmwbymFo3Q1XoMOyCnKODX1H-BtkmxZV4Psvh_eIcHXWZYUkx7V7a1Ow_FXg-M619pSZkbQUyMvzBzgl2Yt-RTAGNaHCxjsY2XcAPE_BVgjZ1s4xOH_2v4npYKIbcapS-Ab5I_uNndtV-AjZrtsG0wEZ_DNlczbllxQCl9szl1nGywcd-AbZ8K2AaPwnH9tWWkG9IEPsF1324b2ZFsQwuk2BXvjbTuc821pQ6_bAj4621JQ4W0zAnOy7XECMFLafBt7sLepdBwBTVwCMJ4jt6vBU8o9B5swe7sainA9z92GNJwCdmLDZw7ml8AvTGzbgJ1uC88GQl3bBlwwJXKbesIT-NkV28KmiMsj8NnxEBCML9BrSwCiDiN02zCAS48aeMkBngIt3GGCAs-4DUO3uQNAAlgBv1DqboP01b8YereZTSl-la75kSO1LpJCbEAChBPzO6CiLlIOnsDbFoLDsGAakJcOBdLNPDHPQ7bDUFwzFsMj-IUDMY6hEfjKiIszCi4IUApmOAJdecAFAbMG82s7MADmOZRtO44LLMZJkILAJAgmCDPUApwRFC7N4CAnAtwe_LvNOYwRuEBhKJJSYCtki8Bth6NIMOCTMfjbtTQZUeEwlzC3MLvbTDgwCOEyamYekHMH5mKbAZkcBw7TIhjD4XqAnIGMEfwLWGCyKWJ3BUwRjJ_C765hI4PplhRaoYjCvEAG58JnSXCOpeAwDop4qPQAJdBrbzOHQENIqz1kjCslDJJATzbyGX6FBg5FTASkwKZA75ZVgiY2loAYq1d3UPNso5TfvPl4eWY-fDMoy9vi9d7e_f397q0qB1mkHyCM3w2y4d4tAIE67xgHuINo9_rwz_7N-UHw8CUYFM4h6Pr7k8trlt_Jk49R8fn-1FWD_P3-4GFIdvSvx5dXX24HqT4ud35m738-y53fgtP9K-r98BDc-b_dnsfuzuHwOxIdH55-u_Pt-8Pzq2NafhHfPZ7F13fX6uDBuWNBcX5w2v988ivZOXKcj0fR6cXI92JvpzyL_Mvrz-EH--efjvKjHZ897H_6-fTut9A5Pfxw15dnP337XXEW7ve1Gny8-eUmf_8u_vJQRt9d3f_4bZEfBVcqLMjZT-c_-N7OPSuO7n_87afk6Ir88i27OuLst4vL_NfvP4Vqp3Tvj989PCTB8NP7zxe5fke-_PLzB03Ijnd2HpyFhz_JneyX0-PsHb9WP_Nsp8zk0fl930nfCf_IV_aP-d3Bzfff7-9_Ptrv99M-UHTa7w_7B9nR0f33_X3PeXz4_PHhkSt75xd3UPpk_90HyFkOvjyMwAr_ODr0buOPb83k7Y1n7021wlZUU1l_M5bOWFlQ6i0L4qshfqvj_afN0LpNm8muZsb-TduJznZoIaftaFc7o9rTdqyrHZq9abM6FgAeNEl_g8F0HoOBr3GMv1shqMBJlkJ4pkPrWpVllmHcVg60dQLOGFew4sD4u7zcY4f41wR9p_BjMVaAYu8kTvXnLFcX1Q_4lO3epjdbk2EaDdyqHWDl1U4x7Wxi_nhqfir2TtNjNQRlAr9alLlKriF-SPZq5OhIq5E-6Xf_2HRaEfnNK2qVmWWWX5EmiF-sQ_0ANMXgzKvQ9VgPi6qxCZCABwBBLUhA89gfQeT9PiuxglezpmLe3ph785jZ5uEnlajPmfWDggbZirycybXeAwPbrDSm8MVZWQ0VHvt6Ljf7YVhYhqOONRmiVdUO_hp8MQHEV7484Qu67b8bW7pskipH6U22GmNUEEDOFDzGaTmjQBgH_bG2iP63FadBrlUBNDUWki3oPg5inQaPSKMho_iz-WNCvL83f_4ItcKw_6u5ecoXzOD-8nyZfjUxEcRLkIeb2gT2ih_A9VfP8MfxlzeVBFpFlpcYPOw_Hl5cvcPVj-l-D1M-qzN9bH2ly6qW0oR5u1XmI401kkiNEvz9-5FKYqxakOavZ9XmtTTLh5MFYEAFIS_WriuM14-3mLr3z86qJ_2krJFhd-MqRFVtqAdkxeG4AlH_aHak9aejPlBJUJhxg2ImI5jbtK76NeoqPo4Nd-NhkBSOaSqSDCuLWfhojRcwZpBP-v7mDQyxhj9OMl8lbIq-Ip1utZ7TSS9GOMNx_W3L-q2iGX86rPi3Zd1UYfG5LlWoSrV3CnFysYdsqYQbPs2AG_7PEGQKU2YkKUhrVb6pwfZWJYORZXTUE4oLsyUWvtelZyIPsxhWJWkM9_I0HYNlh8C73hinp8thxQZkroB0VcprVNYYl9VE9vLsqNaarrOqvL8BCzoQvd1yVyIbLdaoMIt-KxMpl6pjAvgPVTF4vkY-wbCyUiKkZUAneTAi_T3t3A9a3UKgwqyre9S539PS1RMPvi7GSVOJ8Q4byE8XplX5fTS8xcWVP47ZJzoZ6vL35fIF8EYlz5beGfC3W95KrKzB9l5aWA5UUX5ID8G-DTYxNHOwrCokCGuBhlTQL2hNDzXY7GG9JPPs-ZqPZVXi2tAvSNwJLgA9m6g29KrEVFB_mDID6XforDdRZm8ZHz8piJuzpNTp1UAnz1frLjyr8hbhrQqBVWHYezki_2Kh499DdlYwMCZ9xEQRV8r7o1y1WNvHFW_QfM1WMzTd2HCD8Up8Plcw-CzRuFXpd4hLq9MQV2Uef94oLJ2LZ2V_UUFbNfgfHcjRzQO55QyfbICp97880yh1oFk5fJvZhrP3lcB_CwK_KstXVn9V26-yNK8om5WbRU1iGYf3c5WGdSzwbP7280ClcVEaZGuzdwxtVeAvKD5YYBoqP8bRbFTfeoJknRLXBPiFKdM_DOLbzcp2LQzrle2uRvmN_jcuqG-m3lUvq3mK5ym44dtVaU6FGlqQw2Nm05cwrI2d4nQzM8v-dEdSS8elSm90WJ1YMTutNxCzblyrCty-SvD2CWVV4CuTy__mlee5JNYas-aEHmoVJo_9OMGjJJssT3UgmpHdeh7mlxYRgTXBsPdiLroeZH3so6pLXCSjTajtxrWy077Ho746nJxGqS-T-Hs4BPZiS6zPE9srFcbFJsZnBkF71hZIaQ33xMXVH690aXZVVPsm8KNxcqbFaXo7Kg2qt1vDuAh-8UdRhPff4DGh3Nzuc_Tu3dHB9emno_rEShPEsOCXdDT0cZtq9f_pTStX2pxmtIqRX1Qf3259ivW9GcghsCJOCqQoSdRt0aARN0bUI08AbgE20-okntyXMx_XtEE3pqMHnZdA7w8qD_JYd45r8nzJoKoOs7rS2IUNL4TpRlS5IVy7uVRlJ47qIp9uLHgdTic5-HAB7OnwViWdPddPl3CixK0xIKdxFAd4MmrxlONGmqrVAr7Uu-YWzHd9lLQbx35irj-aj6B62A18kcUFLlTNh66fLuBqvVWgg6vV027wQx2oTtqrh93AkxPiWWoOiM_HMmm1ANP7LDVCDkoz9bDz0R0luu1C5yL8UA50Xp-26sJ0DjZq3GSh4lSbxzvxNFos4JW5zqKDQ_isG7S6paGDBny2wBK1riroYGizTTeq6nR_pyFbBFptLezkX32eeMEU1LvOOthfPV3AhPH1AR30148XKImxv_27LA6rQ-Ud6jLTbJHByILPm6MxB-Q3RzN7dH5zjO8SVXzunO_6aTf4x9KUQ-ZgqYKllZCgUm2GAXVrMwyXs4HEFPZycQhh7BdEfLgyuNCATdosmN9ylB6CkysXzO2KqMyw5ivClLS1cFWmfC6la2OsBLS-YmiRDFdNliACX3SyINpZDdPkNoMTrZJycJFlyWYIn1yltBGdWVmoNDzE61g2JBRvcxndArLxyD7MC1enUzqL9c3eOJEwR6sxtK_PpV-VOVbIvmTZ8CeTxuCneh-1Xe-dhiDvMAZW50ZExv1gwx_Hp_DfmDSp3siNn8f7uEeFrq5Zq8pt5ufGFmts2txefanwwNlr67J_edT7LkcCTF06GPQ-xXkcxiq1vtX3Oul9TOPfRto6PXxtaUcxIjgNXE6k8hUhithhENq2Eh7lyvOVlpwHTLm-43OuIsfm2udURJxrGfTMKAzRry1H9CBaTuIApPO1RXqvqI0b5YFPOr0pBz1KmqcvzB0c453zvPmkSgksE_P33K4n1n1cDqz97N6c03gD-Zypa1k5_sUrO8VWzZ46Z-xswZa2sJe2GF_jYpLe9vyw9vx8fH_6_cejHrjaLEyzEqYH8pMhBNNPZyeIhBYODwSTRAgpbBq6jktsFyQKZoozz_WiwBEykLZmVDAv4F7gBTSSbqhsuzU7gvamhw9fW7Q9VxLmJovq6bBKIz1Rng0tNNKmTII2x4oLq74eCdCowkJnthL75zHGniO450onkJRDxx9yTL_TsneGN1FYB8kIbyp4yqSIOCxyqGaepD7xiRuSgMjI54HSigQu9UJtE1v4gRsJ3wupK_HgeehyxaIo9FpMkk7PfLrUv722OGnyiPf-GeQqAv79yxwCca36gMj4tNDkMbOgAfCn_dxSua5Gb9VnZlsIof3VEM-OzkDdAENLmC7W1IN6mqYa8EwEaAstNIYTVGNiFiAEE5SNsbb0FuMSy1yhpSunbmFK36Pz2IGSBIZVgUBmo6K7zbnKy1gluFH-HqLA7obXenhrpNJU4AdahX8lu7CghVjaQi5t4XQrGZ-jZMeDbJRYR0WBXqlXTXKnfvmagr0hKqLgDdxAej4PVehFWlMvcoQC5xEExAulCCPQt8jRxA-k40puK-1Fqu0ieEO_hNvULzajX6xTv5YLu5wjlkvO-q0o8K9cdGmT892rgk20QltklxpTi6bTApG1Kie_CNNAwWcmmkRV8dsCvclAb6JHNNx9LEyV_2bKME_UxVxHe6ni1MwzSBreNFB0-1uuXQc8LPoT6UW2BhkO7MBxuEd8cBchpb52A0V06HEvJFQxqal2SKgkJyJq-1tnob_l3n9bOV6Fh2drcRFtHAmZL4UVxeAtmKUmy1fNo0K9gyzPR7eA-oVmdR435ULvfPSoe-c6jEfDbsshQkltCgEMME5G3Ga2AzGj7YLJ8CQPHc2U8hxBpB0oBkEl_Bc5XhhRLV1w0m3P7NhNy8Gb3LRnLAfvtByd6qJanvn5jrk1lauaFNY2KZ2DnJxetY6CQdbd7irR-hbv2wLTkpZYyP8PUH5nrvJPj-geZHHSO0QvgNt14ht860IKc2SKdK8tj_H68z5MZH3VK-ga6Oqu47m9uhhXNay_zG_ZUIAwZBF1Q0_jXV8S4ncvdH1ugyqQKPBsyLoc2w4h3VK-DCJbKBH6jnKZENT3JIlms6v6tC0IHe3VMeRr63jncud4Z39nv6EfkrStzfTUMmNPji3jz1ViBeEVaZ2QrzhiAa-smubeK8dBgGGV7lcJwA5mD3gHygRzVTrFWkxPVJnF-BbQcacmuTA5RZVsmGTi6Xnqv43UzpNJty2T5_3j0wOw3SAu6QiiXuv7UQxMjxOMwk3Nbnxg_vRmBEFLU5TAJkqfQLYnNCTqLmMeUbYvwYwSypTHJQkhueGCKMEjkLYA0qFQ0VCCWHlB0I7CXDYVJUY65YZO7eBHlAaIqlOrvhgQnJcKBlY0SpIeYzN5PeZr41t0ezbELcYYjUt90MVwlOLlOCAyZhUKpxxlbOLWLJMdW-EIV2fHsc6fOZPeXOtyPcAbe4Ywxt5FHhdDIC-wLvFr0wvqUNhh4FMSCA7_QkwsOPcV94XSECxLHShHEY9DSh-EtuPISMPUCq0iQiH4eBJUTGbLJh3x80-otOe61Blkd7h9pbACc_J49gbOCBrYVmEC0ALPzoFC9l7RSpvRbU3eNdPQ6KLHSFcyahQap3K8FGYi3He51l_0Nt5jGICM5-P7RjEWT_XwsfcKg9uFXdqtLqdtakuBQw7MciiiafeHw6k6XIKlImFcY5pcAfJvniBSMle2z1WSpfrx_6jbrPj_kE9oZC_IeO-DOYqEtgvUuynoNpcy5A4NiPTAgXEfBFs6UcAjShzmOw71XJ8KyUJli9CJWMSIVsyPiGuDXWtHz66Yerj9nWPrslmXEW0rdaLQudWhG35hkwrLNXj6G5BAVf-CpmVc5Kjs2WM2Gk-5mX6FDbbHn2mtHGCasiTM7tPuemTvlfCeeEbRFMt9SEBCI_EQktXa1pS5_4Qwjc6rVp-gQH2KC9RkYNKXLyoHjufI7kncZXPWHXcJykSvWi-2qgVj8GW011xBfgokW4VwP5KR72sSEcmUKxVxfaao73tMc98ObJ97AYiuYwvh-ZJQKRn1haMDh3Fiq85QjbTkGGO1hiS3K-asitNcbm6Ho46Fl26PLZyx6VW8hKa8eja1kBg_oREcQAwF8l6C0OGSAoyiuoy398oWxER69b4SrM2hp3rFSFNqW0wEV8DEE5lu1DJcOi1jVIoC_TjdseDEQzmtkKGe1_GQ_tPKdgtauAs0ic3RpE9ZXuoH67t0FHxOdO_9yIcPVnWIdapLkGB065JHaSuJcUPtSeFSFRAuIDdXYLkDKZwoConSHqGccQhDo4jaRBIHTLlkXDAaUptrz_fbmuG1Is-JalzuXKJ6NIMbu60ap2kEX0oLb7q3jh7AK41yXRvRbbwIOTHFrh1KjfiZZg3JqwKAwpj7uEZV3IL8masYM1zynVhmNMkxqI9VvRILg3LKUBeaxny8L8EyuxQaKT-6H0y03IWJ1ivOxTx19OgCxXjlPvUwr7jo1rcNNenfQ0_suVmYGkEuApN7GN_Fqa7MWCsHOwD5atXBpUO5HYoIgnUZOtpXShHiRopGTPk6tCUVjh1QTVQopeYeJaACKvCd0GW-q1fMwGbWUCmGrMMsnyZUVhzh0jCmY6CxqAFnENEYWTjGglJhNfOrap-eCfQpm0T6bRRVm9-1mEj5vEk4wV5BgW8GfpanJjID5l_qFD1_g_USbYznCdfxGFEOUQovt9aEM-kF3OEiUC6VOlJgkXzbhYyYOMqOVBC6yvFUO4USfK5zPgbnfGwd4_-bhcaZhJitnUBUpReJc2Gz2SpIj8rOcPKP8oJzp0vMm64-WjzwE4EeZwUocJjItRSnnw_BbiZJ1iqFeVHAfOZLCvoD-W8UClcRxmzpR9SHGIo7REsRaOqHOoQphUBMMccVeMU7sWlbe3iX9jjPqV_gJIxr8ePHGNGAivSEN6ce9nvUJeZOg5y_oqGRRGUd5uomSyPAN56Pdwl6gd6H9PHB6g9HCeRDjTnwAjsQIhRhIKNQSUaYE3igPpRBpOs4UkYKItqQ-1SSUAZcuUFIHBfLEVREdrse77bq8bJrJQ_EIFDoeH9QZTAoTTECXKkYZ_vTPaoQOT65R3nmXFV1j_IrypcUCyhf65rPegMO5GaIv1pto_ZUJqbXNAOxIwgsTGXjPyFZc-aEmBe4nnZ1G3_WPRNkVTPztBzgBCRQJAhUoEng2j4k_pzRwFMSfJrrch1J33V8iCptart4CzlEkWC4hbapb_sLpG22StlYGbNCDbLA7NXrU-gZjSkocRsE2K_HSgDAEOAlm3VEZ2rntvekit2I6OpNIZi6Mbkk9rPnp2Iuexre2dII-0y6B2mb1x32vfTy4N-8wDW_DD-AYL8ox5bzapTcDkYzZfgP8KwYDVu7jVjAIu67AY9sFuDbdSHygGyIU5BwAuEh51JQFXlaEl-C1-O28LnGl5lILiNnRT8mOuzpQj92YGq8BchKCoEeJujhuNALpg9Ae_-M8OU0eKHzv1Cu2mX5xsM5EWTb5zGn22X-xW3a_FL-4aMyOyQPFYSeJ1rdPVr7Omm7T46RixsK23MiW3lMKBVxB35iDg8FcT0vYCKKIDO2GRi3yBHgSYWP-wOU4t6CjTCsbdBe2by1cfKVQ-YYBmduovvKducajGrTSR1h1jbJnICZbuSz1A3YvKKEGQW_qCelyrnJc7NuWVndGimHEDkybxXqHeLPafYk6v1qlcyJyzm-NVE3IG2XmQp7VwN1k2vcxIt3aTSKN4sKoZJQrymzikSaQ9QtbM1CCOGksgMsXwpCIP4TDnXhd4rlHVc6IYWYL3AgCqfSCbiEbGu2Jj-3eLOPq887zQq9mLFf8wSoKTNiIjKtTU7jN-2NC-wY1y2rwLyyabdjHD-s9h43HjBvQb_tLR7th5hKI1UYM6SVMyktH6cNi_uQs0PL5HFqwF_JpyWcr_XOVeo4jM413BdZAj8E1qEGnSgrT_bhFhOCmXVYqpmyHUJ9JUKP-I5ru1rQ0CahUEIxyEeFq31w14RAVCoY9SmnHELYkGJD2VaF5jqsOxOPUrFwrZK1SnzVeWsLz6gaE1ndRmdVZzshzHPnW3nWYeWdVuRbu3Zj-ccBb73oinb9TEel4ZO51-M1YLtrlEEhzO8XAxMgi0nlaLJmVi_hQpg07F2iri1E9C7PwK90oKqWaytUX10DEMFWq770h7oEz1q2g9YLiBlGcat-pijF_QQKUjLCifJxfwjXHIssLIioFgTslPAjjq8mZJEKQ-ILz6O28vGgQVv0nUbp0mvafGemcrmgqrJ4T0gtD2ZHCEYi69Rbfu-zHHbndurre9xP1ixmNqeAUTQqkPo6juc6uEzCXZ_hrg_l-65tR9qHmBJMDu6KjBzmRjYngoIn9rlEJzSzF3JeCfNy5wcsYsLfy0UlTLZuCXPmBVeTlfSqGsPmP9wfl2uoN8fczaa-bZN5VtkTs_rS2LxJ__Ra6Z-StuBrNLrPgrXfsAHW17yj7cL_eHmGp9OqKyzGt9hO3gY3obgLYnrF0-TNcPYyGBOoWn3_sSjUeCutJaYYyFIEkHHNwrM14Cs72HzrnbsqnebSw9m--Rp9TxBtQP_kiv_NccwnSL4AQfIFCHoWjnkE0XWka5542i_AEfZS1Kwjb9UbCjbixxxtkxuPgK2NoaGu9Hl92i81AeIFpIG-gH7QFxgHX1MUGtPgbWRk2KY6yV-Ag-u7jcbcM7YRA-xNGcA21OL1BjB5fdYmUlhFGJtgmK_aYmMM8qWcJ30pKzN3es2j0xCwzL7Ct8JYnU9qvNV33ONKwt0YO7UX9Vm9hbjZ5fS9xCv2OH8a-MbCJF5GqtdRrmoNvcE7-VxrbL-ATbNXlJvqhdPNOZy-gnp2DueJXfWG6ZbYTV86vQKC-pXSrQFM3zK9ohTNs2t8Q7tIX0aCnh1uNARpRafUoJiK1UAa8r1q9tMAoc5zGCs29XgvNDP8pUz0iyUo9nO9T2NK-DJAU3ltTKK3SDurF7k3lXP6avcVdXO-fWYvM4cv5q9fIvPmz6NpCuWsNHfNMgt9HvdfKOYTG7vpzR29XNeaNLj3PPPztMeqBje-VdbsEDOXymZpFN88uSJW4ym202I_K4rpnbIXcZqqYHJ3aLM9VvLMuX7Ad6fzG_OeJcvPskSrtL4lam9-Lwdj0JO43H_E9Z_xvbDLwEeFrl4mW1fjl0NMhmlO6dW7ApaDmTMoRX0dLd51OuUJrmPNgcALeT-kRaJGN4NyjXGdqDtd32D-DOZNlohX5WCbH9UKWLgGmDmPWK1iTC8SZmJpTwZurZ72tUYax-vP8yEvEhXoAUiQzptMqu7anYzPIWRxe7OYXt-VOYGylwA94f2EHYzTJf3lk6tvJ1BiMQiK3ewIxTKyGqq1xuDglwu8_m7CiCV8qG6lH7d2-XL06xIyYfVa42q_fn0NtqE0zPLN9eTyCVpneFUHl1mSmJWUprQuBhz758mMckew5WNbXxDGe57Wh5xcCzu5apxQbi_XidmeahV8szd2Wm_2LlQ5-BDtj-IEz9z_o_e_yWXTZA==
```

### Screenshots

i don't think it's necessery

**Comment by Paliak** [comment]
This is likely caused by the explosive arrow skill inheriting the totem flag from the ballista.

**Comment by BillWiegert** [comment]
Same issue also affects Storm Burst totems. As of version 2.32, PoB incorrectly includes totem duration mods (Pursuit of Faith, Primal Manifestation) when calculating Skill duration for Storm Burst with Spell Totem support. In v2.31 the duration in PoB matched the in-game tooltip.

**Comment by LocalIdentity** [comment]
Fixed in 2.33.1
